meeting_notes/2018-02-02.md

```diff
@@ -0,0 +1,34 @@
+# Distributed tracing workshop
+
+This draft was discussed at Distributed Tracing workshop. There was an agreement with the format, scope and semantics of this specification. 
```

Comment:
maybe add link to Adrian's docs, with meeting agenda and notes.

Reply:
fixed